### PR TITLE
Add lualine.nvim configuration example to README.md

### PR DESCRIPTION
**Pull request by eggbean**
This adds a configuration example for lualine.nvim. You might want to change some of the text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,23 +31,39 @@ statusline like this :
...
 
-Use <a href="https://github.com/dhruvasagar/vim-zoom/issues">Github Issue
-Tracker</a>
+## Contributing
 
 ### Contributing to code :
 
-- Fork it.
-- Commit your changes and give your commit message some love.
+- Fork it. Commit your changes and give your commit message some love.
```
Comment (dhruvasagar) [inline README.md:58]: Lets not change this

File: README.md [hunk 2]
```diff
@@ -31,23 +31,39 @@ statusline like this :
...
 
-If you're using some complex statusline plugin, you may need to refer to it's
+If you're using some complex statusline plugin, you may need to refer to its
 documentation to figure out how to add this to your statusline.
 
-## Contributing
+#### Example configuration for lualine.nvim
 
-### Reporting an Issue :
+```
```
Comment (dhruvasagar) [inline README.md:39]: perhaps ` ```lua` would be better

**Review by dhruvasagar — CHANGES_REQUESTED** [review]

**Comment by eggbean** [comment]
Sure. Sorry, but I'm not sure what I am supposed to do on github here 😳
How can I apply your changes or do I have to do it manually (ie. amend my commit)?

**Comment by dhruvasagar** [comment]
@eggbean np, thanks for your contribution, I will make some of these minor changes myself.

**Pull request merged**